README.md

```diff
@@ -71,65 +75,167 @@ is loaded.
-+ `color`: required, three RGB values in `[0,1]` range, in linear color space.
-+ `instances`: a list of light instances that use the settings.
++ `strength`: optional, minimum value `0`, defaults to `8` for `sun`
+  and `25` for all other light types.
++ `color`: optional, three RGB values in `[0,1]` range, in linear color space,
+  defaults to `[1,0.8,0.638]` for `sun` and `[1.0,0.88,0.799]` for all other
+  light types.
++ `angle`: for `spot` lights only, optional, in `[1,180]` range,
+  defaults to `140`.
++ `photometricProfile`: for `point` and `spot` lights only, optional,
+  path to IES light profile file.
++ `width` and `height`: for `area` lights only, optional, both properties in
+  `[0.01,5]` range, default to `0.2`.
++ `size`: for all light types except `area`, optional, in `[0.01,0.5]` range,
+  defaults to `0.02` for sun and `0.1` for all other light types.
++ `instances`: required, a list of light instances that use the settings.
 
 Each instance has the following properties:
 
  + `position`: required for `spot`, `point` and `area` lights,
-   `[x, y, z]` coordinates.
- + `rotation`: required for `sun`, `spot` and `area` lights, `[yaw, pitch]`.
+   `[x,y,z]` coordinates.
+ + `rotation`: required for `sun`, `spot` and `area` lights, `[yaw,pitch]`.
 
 [More detailed description of light
 properties](https://www.shapespark.com/docs#lights-tab)
+
+### `sky` object
+
+`sky` is an optional object that can be included to change the sky
+settings. If `sky` is missing, the default sky settings are used, if
+`sky` is set to null, the sky is disabled.
+
++ `strength`: optional, `[0,100]` sky strength that is used for baking,
+defaults to `6`.
++ `color`: optional, three RGB values in `[0,1]` range, in linear
+color space, defaults to `[0.855,0.863,1]`.
++ `ambientOcclusion`: optional, an object that configures ambient
+occlusion parameters. If `ambientOcclusion` is missing, default ambient
+occlusion parameters are used, if `ambientOcclusion` is set to null,
+ambient occlusion is disabled.
++ `texture`: optional, an object that configures the equirectangular
+sky texture that surrounds the scene. The texture is not used for
+baking.
+
+`ambientOcclusion` has the following properties:
+
++ `factor` optional, a float that specifies how strong is the effect
+of ambient occlusion, defaults to `0.05`. `0` is an equivalent of
+disabled ambient occlusion.
++ `distance` optional, a float that specific how far to search for
+occluders, greather than `0`, defaults to `1`. For example, a `distance`
+`0.5` means that if there are no occluders within `0.5` meter from
+a given point in 3D space, the ambient occlusion has no effect on this point.
+
+`texture` has the following properties:
+
++ `fileName`: required, name of the file inside the archive that
+stores the sky texture.
++ `yawRotation` optional number in `[0,360]` range that specifies the
+rotation of the sky texture in degrees. Defaults to `0`.
+
+### `bake` object
```

Comment:
This is for cloud API only.

Reply:
Fixed, sections removed from doc and example.